tests/osaka/eip7692_eof_v1/eip7480_data_section/test_data_opcodes.py

```diff
@@ -14,6 +15,24 @@
 pytestmark = pytest.mark.valid_from(EOF_FORK_NAME)
 
 
+@pytest.mark.parametrize("index", [0, 1, 31, 32, 33, 63, 64])
+@pytest.mark.parametrize("suffix_len", [0, 1, 31, 32, 24000])
+def test_dataloadn(eof_state_test: EOFStateTestFiller, index: int, suffix_len: int):
+    """Basic tests for DATALOADN execution."""
+    sentinel = 0x8000000000000000000000000000000000000000000000000000000000000001
+    eof_state_test(
+        container=Container(
+            sections=[
+                Section.Code(
+                    Op.DATALOADN[index] + Op.SSTORE(0, unchecked=True) + Op.STOP,
+                ),
+                Section.Data(index * b"\xbe" + sentinel.to_bytes(32) + suffix_len * b"\xaf"),
```

Comment:
```suggestion
                Section.Data(
                    index * b"\xbe" + sentinel.to_bytes(32, byteorder="big") + suffix_len * b"\xaf"
                ),
```
Which python version are you currently using? 3.10 fails if `byteorder` is not specified.

Reply:
3.12